### DISCUSSION TITLE
Support for locale/custom validation error msgs in Password validation rule

### DISCUSSION THREAD

**Discussion by koushik355** [laravel/framework#37686] [Ideas]
Hi, I wanted to use the `Password` rule. However, it lacks locale support/customizing the validation error msg. 
If I've something like this 
`Password::min(8)->letters()` 
I can only customize the `min` validation error msg from the `resources/en/validation.php` or from the `messages`  method of the `FormRequest` itself.
For other validation like `mixed case, letters` I can't give the validation error message in another language apart from English which makes it unusable for multi-language apps.
I tried certain ways in the `messages` method of the `FormRequest` with something like this 
```
'password.letter' => 'is this showing',
'password.letters' => 'is this showing too',
```

However, it didn't work. Next, I checked the `Password Rule class` itself and I saw that the English validation message is hardcoded 😞
![image](https://user-images.githubusercontent.com/66407490/121924794-dc7e9a00-cd59-11eb-99ca-1ef5724238a3.png)

Can we please have support for customizing the error messages so that we can include our message? It would be awesome for multi-language applications. In my current application, I can't use it since I need to show the message in Japanese as well.
Let me know if I'm missing anything. Would appreciate any input on this. Thank you guys.

**Comment by chuckrincon** [comment]
Hi there ✋🏻 !

This validation uses the translation strings as keys that are stored as JSON files in the `resources/lang` directory.

For example, if your application has a Japanese translation, you should create a `resources/lang/ja.json` file:

```json
{
    "The :attribute must contain at least one letter.": "また、:attribute には少なくとも1文字が含まれていなければなりません。"
}
```
